core/messages/en.json

```diff
@@ -214,7 +214,26 @@
       "deleteModalTitle": "Are you sure you want to delete this address?",
       "submit": "Update address",
       "submitting": "In progress..."
-    },  
+    },
+    "Wishlist": {
+        "noItems": "No items added",
+        "new": "New Wishlist",
+        "createTitle": "New wish list",
```

Comment:
Are we using Wishlist or wish list?

Reply:
After discussions with design team decided to use Wishlist for all cases